lib/orbit/operator_sign_in.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Orbit.OperatorSignIn do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias Orbit.Employee
+  alias Orbit.RailLine
+  alias Orbit.SignInMethod
+  alias Orbit.User
+
+  @type t :: %__MODULE__{
+          signed_in_employee: Employee.t(),
+          signed_in_by_user: User.t(),
+          signed_in_at: DateTime.t(),
+          rail_line: RailLine.t(),
```

Comment:
suggestion (medium): Should this be `route`?

Reply:
Good question. I didn't give it much thought initially but as I weigh it I think "line" is more appropriate than "route," because "route" is already a concept in GTFS and I'm thinking of this as a concept that's agnostic to exactly how routes are modeled in GTFS. For instance, the Green Line is actually 4 GTFS routes but we'd call it only one "line," and I suppose you could imagine a world where the two Red Line branches are actually separate GTFS routes. Of course, when it comes to the former that's not a concern for Orbit, and for the latter I haven't heard of any future intention of modeling the Red Line that way, so maybe it's not very sound logic after all.